gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -577,6 +966,82 @@ def main(args):  # noqa: D103
             )
 
             ht.describe()
+        if args.prepare_vcf_header:
```

Comment:
Your argument below is `--prepare-vcf-header-dict`, I think it looks better if you change the one below to `--prepare-vcf-header`

Reply:
Good call, I'll switch it